uiowa/uiowa
# Parking and Transportation migration

Resolves #3264 





# How to test



Sync D7 site in DevDesktop (transportation.uiowa.edu on uiowa703).
`git checkout p-and-t-migration && git pull && composer install`
`blt ds --site=transportation.uiowa.edu`
`drush @transportation.local en transportation_migrate -y`
`drush @transportation.local uli /admin/config/system/sitenow-migrate`
Complete the configuration form using the help text to configure for DevDesktop
`drush @transportation.local ms` -- shows migration status; two migrations should appear, one for articles and one for article redirects
`drush @transportation.local mim --all` -- runs both migrations
There is an ImageMagick error that can be ignored, as can the link replacement notices (no attempt is currently being made at fixing them, so the notice is informational only)

Spot check imported articles and their content, that images are present where they existed in the D7 site, summaries, etc.
Check that article aliases from the old site work following the redirects migration.
Check that all imported articles have the news-ParkTransit tag.
Check that articles which had tags in D7 have the appropriate updated tag on import according to #3264 
Check that the title and published date are not duplicated in the body content (see example image below).

![image](https://user-images.githubusercontent.com/49074566/116886642-eeafd900-abee-11eb-9dfe-318691bf88f6.png)
